### DISCUSSION TITLE
Detect when user is adding a chain to their wallet.

### DISCUSSION THREAD

**Discussion by nascob** [wevm/wagmi#2013] [Q&A]
I've got a UI implemented that relies on `useSwitchNetwork`.
If I call `switchNetwork` and I don't have the network configured in my wallet - I get prompted to add the network. 
Is there a way to detect when the user has been prompted to add the network?

**Comment by tmm** [comment]
Not at the moment. We could look into extracting add network into it's own Vanilla JS function and React Hook for the first major version of wagmi.
